draft-irtf-qirg-principles-09-rc.xml

```diff
@@ -260,23 +263,25 @@
      distances <xref target="Elliott03" /> <xref target="Peev09" /> <xref
      target="Aguado19" />.</t>
 
-     <t>Fully quantum networks capable of transmitting and managing entangled
-     quantum states in order to send, receive, and manipulate distributed
-     quantum information are now imminent <xref target="Castelvecchi18" /> <xref
-     target="Wehner18" />. Whilst a lot of effort has gone into physically
+     <t>The first fully quantum networks capable of transmitting and managing
+     entangled quantum states (defined in <xref target="entanglement" />) in order
```

Comment:
Did you notice that I added a reference to [Pompili21] (https://arxiv.org/pdf/2102.04471.pdf).  It seems to me that this goes beyond the "prepare and measure" stage.

Reply:
Yes, I've seen. But I was just trying to define "fully quantum" network without mentioning entangled states and I think that defining it as "prepare and measure" stage (which defines minimum end-node capabilities) via an intermediate node (so that it's not a p2p link and actually some sort of network) is suitable. And yes, the Delft network is quantum memory stage anyway, but I don't think we have to draw the line that high for the purposes of this draft.

Reply:
I took the easy way out: mention neither entanglement nor prepare-and-measurement at this early stage in the document:

"The first fully quantum networks capable of sending, receiving, and manipulating distributed quantum information have recently been realized [Pompili21]."

Pushed another commit with this change.